nginx/templates/multi-channel.conf.tmpl

```diff
@@ -46,6 +49,8 @@
 
         rewrite '^(.*)$' '$1$default_rewrite_params' break;
 
+        include /etc/nginx/conf.d/cache-blacklist.conf;
```

Comment:
I get why this way of structuring the blacklisting features is cleaner and more contained but I think it's a bad idea to include the cache-blacklist.conf in the location template as I would rather avoid random, seemingly untraceable errors:
https://www.nginx.com/resources/wiki/start/topics/depth/ifisevil/

If you know that this is safe to do in this case, that's something different, of course.

Reply:
If I put it outside, the cache key calculation is not done correctly and URLs with query params do not get mapped to `/loading` without query param.